internal/cmd/iam/command_groupmapping_create.go

```diff
@@ -0,0 +1,56 @@
+package iam
+
+import (
+	"github.com/spf13/cobra"
+
+	ssov2 "github.com/confluentinc/ccloud-sdk-go-v2/sso/v2"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+)
+
+func (c *groupMappingCommand) newCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create <name>",
+		Short: "Create a group mapping.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.create,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: `Create a group mapping named "DemoGroupMapping".`,
+				Code: `confluent iam group-mapping create DemoGroupMapping --description "description of group mapping" --filter "\"demo\" in claims.group"`,
```

Comment:
What does the filter mean?

Reply:
From the `openapi.html` file: 

A filter expression in [Supported Common Expression Language (CEL)](https://docs.confluent.io/cloud/current/access-management/authenticate/oauth/identity-pools.html#supported-common-expression-language-cel-filters) that specifies which identities can authenticate using your group mapping (see [Set identity pool filters](https://docs.confluent.io/cloud/current/access-management/authenticate/oauth/identity-pools.html#set-identity-pool-filters) for more details).